torchao/quantization/quant_api.py

```diff
@@ -2367,10 +2368,19 @@ class ModuleFqnToConfig(AOBaseConfig):
 
     Args:
         `module_fqn_to_config`: Dict[str, Optional[AOBaseConfig]]: a dictionary from
-         the fully qualified name of module to the AOBaseConfig that we want to apply to the module.
-         Also has a special key: "_default", if "_default" is present in the dictionary,
-         the config for "_default" will be applied to all the remaining modules that does not have
-         per module configuration specified.
+             (1). fully qualified name (fqn) of module or
+             (2). regex of fully qualified name or
+             (3). "_default"
+         to the AOBaseConfig that we want to apply to the module
+
+         Config key ordered by precedence:
+           * fully qualified module name, e.g. `language.layers.0.q_proj`
+           * regex for module names, e.g. `language.layers.*.q_proj`
```

Comment:
clarify that regexes will be matched in the order that they appear in the dictionary